### PR TITLE
fix: some issues on f36 website

### PR DESCRIPTION
**Pull request by massao**
# Purpose of PR

Fixes some issues on the documentation website.

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Dropdown/README.mdx [hunk 1]
```diff
@@ -39,7 +39,7 @@ function DropdownExample() {
         <Button
           size="small"
           variant="transparent"
-          indicateDropdown
+          endIcon={<ChevronDownIcon />}
```
Comment (burakukula) [inline packages/forma-36-react-components/src/components/Dropdown/README.mdx:42]: I guess we can remove Dropdown component. I think Alex removed it in his PR, but it's not merged yet?

File: packages/forma-36-react-components/src/components/Dropdown/README.mdx [hunk 2]
```diff
@@ -39,7 +39,7 @@ function DropdownExample() {
         <Button
```
Comment (burakukula) [inline packages/forma-36-react-components/src/components/Dropdown/README.mdx:39]: I guess we can remove Dropdown component. I think I saw it in Alex PR, but maybe it's not merged yet?

Comment (massao) [reply packages/forma-36-react-components/src/components/Dropdown/README.mdx]: Done

**Review by gui-santos — APPROVED** [review]
🏆

**Pull request merged**